evennia/scripts/taskhandler.py

```diff
@@ -101,23 +147,45 @@ def add(self, timedelay, callback, *args, **kwargs):
             any (any): any additional positional arguments to send to the callback
 
         Keyword Args:
-            persistent (bool, optional): persist the task (store it).
+            persistent (bool, optional): persist the task (stores it).
+                Add will return the task's id for use with the global TASK_HANDLER.
             any (any): additional keyword arguments to send to the callback
 
+        Returns:
+            task_id (int), the task's id intended for use with this class.
+            False, if the task has completed before addition finishes.
```

Comment:
This should be written as 

```
Returns:
    int or Task: For non-persistent tasks, the `int` will be the task ID [...]. For persistent tasks, the 
    special `Task` object is returned. This wraps a Twisted deferred and has a similar API. 
```

Reply:
Done with https://github.com/evennia/evennia/pull/2374/commits/cf8052b816c0011330ead39c85e871c85918a0e4